src/main/java/module/transfer/XMLParser.java

```diff
@@ -44,14 +44,15 @@ public static List<PlayerTransfer> getAllPlayerTransfers(int playerId) {
 
     	final String xml = MyConnector.instance().getTransfersForPlayer(playerId);
         final List<PlayerTransfer> transferList = new Vector<>();
+        if (xml.isEmpty()) return transferList;
```

Comment:
I also suggest to use `StringUtils.isEmpty` instead to call `empty()` on the string that might be `null`.

Reply:
Changed